msal/src/request/java/com/microsoft/identity/client/InteractiveRequest.java

```diff
@@ -51,10 +58,9 @@
 
     static final int BROWSER_FLOW = 1001;
     private static AuthorizationResult sAuthorizationResult;
+    private static MicrosoftStsAuthorizationRequest sAuthorizationRequest;
```

Comment:
This assume all interactive requests will be made to the v2 endpoint? I suspect we don't want to couple these classes in this way

Reply:
Let's talk about this offline today.